samples/basic/blink_led/src/main.c

```diff
@@ -17,7 +17,8 @@
 
 #if defined(CONFIG_SOC_STM32F401XE) || defined(CONFIG_SOC_STM32F412ZG) || \
 	defined(CONFIG_SOC_STM32F413XX) || defined(CONFIG_SOC_STM32L476XG) || \
-	defined(CONFIG_SOC_STM32F407XG) || defined(CONFIG_SOC_STM32F302X8)
+	defined(CONFIG_SOC_STM32F407XG) || defined(CONFIG_SOC_STM32F302X8) || \
+	defined(CONFIG_SOC_STM32L496XG)
```

Comment:
Remove this and add pwm-led0 alias to the board.
Cf hexiwear_k64 board

Reply:
Nice, wasn't aware of pwm-leds.

Reply:
@erwango For this to happen I think I have to add `pwmX` in [dts/arm/st/stm32l4.dtsi](https://github.com/zephyrproject-rtos/zephyr/blob/master/dts/arm/st/l4/stm32l4.dtsi) like in [dts/arm/nxp/nxp_k6x.dtsi](https://github.com/zephyrproject-rtos/zephyr/blob/master/dts/arm/nxp/nxp_k6x.dtsi).

Or is there any way to write the `&timers2:pwm` with correct syntax and avoid the above work?

    ...
        pwmleds {
            compatible = "pwm-leds";

            pwm_led: pwm_led {
                pwms = <&timers2:pwm 0 40000000>;
            };
        };
    ...
    };

    &timers2 {
        status = "ok";

        pwm {
            status = "ok";
        }
    };

Reply:
@galak , since you introduced pwm-leds, any idea?

Reply:
> @galak , since you introduced pwm-leds, any idea?

I'm not aware of any syntax support for doing this.

Reply:
You could do: `pwms = <&{/soc/timers@40000000/pwm} 3 4>;`

Reply:
@gon1332 , are you good with this answer?

Reply:
It's something I was looking for. I will put it into code ASAP and will inform you.

Reply:
@erwango Thanks to @galak's the comment, now Nucleo L496ZG has an on-board red PWM LED.
I omitted the green and blue ones, since I should add the corresponding ST timers, and I do not have time for this at the moment.

Please review my work.